lib/facility_factory.rb

```diff
@@ -0,0 +1,52 @@
+class FacilityFactory
+  attr_reader :locations
+
+  def initialize
+    @locations = []
+  end
```

Comment:
I like that you created an instance variable to store all these locations in, makes it so you don't have to rerun these methods everytime you want to get the locations.